rebar.lock

```diff
@@ -1,2 +1,6 @@
-[{<<"jsx">>,{pkg,<<"jsx">>,<<"2.8.0">>},0},
+[{<<"eini">>,
```

Comment:
I was not aware that `eini` made it into hex yet.  That will certainly work, let me revise.